src/popup.js

```diff
@@ -32,6 +32,36 @@
             var icon = $("<img/>").addClass("icon img-rounded").attr({src: avatar_url, align: "left"});
             icon.appendTo(li);
 
+            var author_avatar = $("<img/>").addClass("icon img-circle pull-left").attr({src: "#"});
+            author_avatar.appendTo(li);
+            $.ajax({
```

Comment:
I am worried about to call a lot of API.
If there are 10 histories, user API is called 10 times (despite contain only 1 user)

I recommend that reduce the API calls.